cli-plugins/insight/cli-configuration.hbs.md

```diff
@@ -7,26 +7,13 @@ This topic explains how to configure the Tanzu Insight plug-in.
 
```

Comment:
I can't comment that far up, but the `Note` links to `cli-overview.md`. Isn't that page a close duplicate of this page for configuring endpoint, certificate and access token?

Reply:
I removed that line